app/javascript/src/components/Dashboard/Notes/constants.js

```diff
@@ -34,7 +34,8 @@ export const NOTES_FORM_VALIDATION_SCHEMA = yup.object().shape({
         value: yup.string().required(),
```

Comment:
Go through the Slack thread once again. I have added a point regarding this validation and how to enhance it.

Reply:
Done